Source/Core/CheckoutAPIService.swift

```diff
@@ -35,4 +36,15 @@ public final class CheckoutAPIService: CheckoutAPIProtocol {
     public func createToken(_ paymentSource: PaymentSource, completion: @escaping (Result<TokenDetails, TokenisationError.TokenRequest>) -> Void) {
         checkoutAPIService.createToken(paymentSource, completion: completion)
     }
+
+    public func createSecurityCodeToken(securityCode: String, completion: @escaping (Result<SecurityCodeTokenDetails, TokenisationError.SecurityCodeError>) -> Void) {
```

Comment:
For public functions can we add comments or documentation?

Reply:
We don't want to advertise this as a function that we want anyone to use. We want all merchants to use our UI component directly. This is discussed offline under a Slack thread I'll mention you now @deepesh-vasthimal-cko